### PR TITLE
Ability to upload from memory

### PR DESCRIPTION
**Pull request by zhuharev**
In my cases, I store images in db and receive it as `[]byte`. In this PR I added ability to upload `[]byte` image without usage filesystem.

### CODE REVIEW

File: goinsta.go [hunk 1]
```diff
@@ -631,7 +625,8 @@ func (insta *Instagram) UploadPhoto(photo_path string, photo_caption string, upl
 	if err != nil {
 		return response.UploadPhotoResponse{}, err
 	}
-	if _, err = io.Copy(fw, f); err != nil {
```
Comment (ahmdrz) [inline goinsta.go]: I think `io.Copy` is better choice than `ioutil.ReadAll` for uploading from file , So please do not change `UploadPhoto`

**Comment by ahmdrz** [comment]
BTW , Thank you for your commit , Please change your function and commit again :handshake:

**Comment by zhuharev** [comment]
@ahmdrz, I'm changed bytes to io.Reader. In the previous implementation we had to open the file twice: to write in the multipart form and to obtain the dimensions. I solved this problem using tee.Reader.

**Comment by ahmdrz** [comment]
Thank you.

**Pull request merged**